README.md

```diff
@@ -1,30 +1,41 @@
 # Deep Security Smart Check Scan Action
 
-For scanning your images as part of your CI pipeline using [Deep Security Smart
-Check][].
+<img src="Trend-Micro-Logo.png">
 
-[deep security smart check]: https://www.trendmicro.com/smartcheck
+## Scan your containers with [Deep Security Smart Check](https://www.trendmicro.com/smartcheck).
 
-This tool is used by the Deep Security Smart Check plugin for Jenkins and can
-also be used as a [GitHub Action](https://github.com/features/actions).
+This tool is used by the Deep Security Smart Check plugin for [Jenkins](https://plugins.jenkins.io/deepsecurity-smartcheck/) and can also be used as a [GitHub Action](https://github.com/features/actions).
+
+## Requirements
+
+* Have an [Deep Security Smart Check](https://www.trendmicro.com/smartcheck) deployed. [Sign up for free trial now](https://www.trendmicro.com/product_trials/download/index/us/168) if it's not already the case!
+* A container image to scan in any [supported Docker Registry](https://deep-security.github.io/smartcheck-docs/admin_docs/admin.html#supported-registries).
 
 ## Usage
 
-Add an Action in your `main.workflow` file to scan your image with Deep Security
+Add an Action in your `.github/workflow` yml file to scan your image with Deep Security
 Smart Check.
 
-```main.workflow
-action "Scan with Deep Security Smart Check" {
-  needs = "Push image to GCR"
-  uses = "docker://deepsecurity/smartcheck-scan-action"
-  secrets = [
-    "DSSC_SMARTCHECK_HOST",
-    "DSSC_SMARTCHECK_USER",
-    "DSSC_SMARTCHECK_PASSWORD",
-    "DSSC_IMAGE_PULL_AUTH"
-  ]
-  args = ["--image-name registry.example.com/my-project/my-image"]
-}
+```yml
+- name: Deep Security Smart Check
+  uses: deepsecurity/Deep-Security-Smart-Check@version*
+   with:
+      # Mandatory
+      DSSC_IMAGE_NAME: myorg/myimage
```

Comment:
`myorg` is a little confusing here, especially where we use it again on the line below. In the Smartcheck UI we call this a `Registry Host` so maybe `registryhost` here?

Reply:
You're right ```myorg/myimage``` looks confusing and maybe out of context...

Maybe we can change to ```registryhost/myimage``` intead ```myorg/myimage``` what do you think?